Avatar/Avatar.css

```diff
@@ -0,0 +1,56 @@
+body {
+    font-family: sans-serif;
+  }
+  .wrapper
+  {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;  
+    gap: 2rem;
+  }
+  
+  .avatar-container {
+    align-items: center;
+    background-color: #f0f0f0;
+    border-radius: 0.3rem;
+    display: flex;
+    gap: 2rem;
+    height: 150px;
+    justify-content: center;
+    width: 800px;
```

Comment:
Try not to give width and height values in px.

Reply:
Made the Changes. thanks!